.code-samples.meilisearch.yaml

```diff
@@ -255,7 +217,44 @@ geosearch_guide_sort_usage_2: |-
   };
 
   SearchResult<Restaurant> restaurants = await client.Index("restaurants").SearchAsync<Restaurant>("restaurants", filters);
-  foreach (var restaurant in restaurants.Hits)
+authorization_header_1: |-
+  MeilisearchClient client = new MeilisearchClient("http://127.0.0.1:7700", "masterKey");
+  var keys = await client.GetKeysAsync();
+get_one_key_1: |-
+  Key key = await client.GetKeyAsync("d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4");
+get_all_keys_1: |-
+  Result keyResult = await client.GetKeysAsync();
+  var keys = keyResult.Result;
+create_a_key_1: |-
+  Key keyOptions = new Key
   {
-      Console.WriteLine(restaurant.Name);
-  }
+      Description = "Add documents: Products API key",
+      Actions = new string[] { "documents.add" },
+      Indexes = new string[] { "products" },
+      ExpiresAt = "2042-04-02T00:42:42Z"
```

Comment:
Sorry because I didn’t get it before the merge, but here I am boring about these dates as strings 😝 

If this is not huge work, I'll ask you to change these types for `DateTime` 📆 .
Or of course, I could create an issue for later ;)

Reply:
Haha indeed, this is definitely a good change!
Should be done after the release for v0.25.0 (the main priority)
Please open issues regarding any changes you want to introduce to this repo 😄